lib/processor.js

```diff
@@ -12,6 +12,9 @@ var assign = require("object-assign");
 var remark = require("remark");
 
 var SUPPORTED_SYNTAXES = ["js", "javascript", "node", "jsx"];
+var UNSATISFIABLE_RULES = [
```

Comment:
As in allow rules inside plugins to specify if they should apply to markdown or not? Or configure all of the rules inside plugin in a specific way? If former - then no.